sqlglot/parser.py

```diff
@@ -615,18 +624,20 @@ def _parse_statement(self):
         return expression
 
     def _parse_drop(self):
-        if self._match(TokenType.TABLE):
-            kind = "TABLE"
-        elif self._match(TokenType.VIEW):
-            kind = "VIEW"
-        else:
-            self.raise_error("Expected TABLE or View")
+        temporary = self._match(TokenType.TEMPORARY) or ""
```

Comment:
why are you doing or ""

self._match returns None or True

Reply:
Oh something dumb from an earlier pass, will remove